src/azure-cli/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -277,7 +277,7 @@ def get_default_policy(cmd, client, scaffold=False):  # pylint: disable=unused-a
     return _scaffold_certificate_profile(cmd) if scaffold else _default_certificate_profile(cmd)
 
 
-def recover_keyvault(cmd, client, vault_name, resource_group_name, location):
+def recover_keyvault(cmd, client, vault_name, resource_group_name, location, no_wait=False):
```

Comment:
will this be a breaking change to user if no_wait default false?

Reply:
> will this be a breaking change to user if no_wait default false?

@yungezz No, it's not a breaking change. The previous behavior is synchronous as well.